get_azure_share_sas/action.yaml

```diff
@@ -23,6 +23,10 @@ inputs:
     required: false
     default: "+10 minutes"
 
+  directory_type:
```

Comment:
I think, this argument should be described in [README.md](https://github.com/SneaksAndData/github-actions#readme)

Reply:
Done :)